src/plugins/template/src/config.hpp

```diff
@@ -47,6 +47,13 @@ struct Configuration {
     ov::log::Level log_level = ov::log::Level::NO;
 
     ov::hint::Priority model_priority = ov::hint::Priority::DEFAULT;
+
+    int optimal_batch_size = 4;
```

Comment:
`4` looks like taken from nothing.. Let's change to 1 to avoid of misunderstanding

Reply:
updated both optimal_batch_size and max_batch_size to 1.